### DISCUSSION TITLE
Can I ignore alerts that are fired in a period of time?

### DISCUSSION THREAD

**Discussion by emiliobucha** [jertel/elastalert2#836] [Q&A] [answered]
Can I ignore alerts that are fired in a period of time?

I want to disable send an alert for errors that occur in a certain period of time. This is possible and razonable to implements?

To give more context of my trouble:
I have several services that communicates and makes requests each other, but in a little period of time when all are been deployed at same time, some depreciable errors are fired and I want to ignore those for a period of time until I finish the deploy.

Thanks

**Comment by nsano-rururu** [comment, marked as answer]
https://github.com/0xSeb/elastalert_hour_range

**Reply by nsano-rururu** [reply]
By the way, the standard feature has the ability to limit rule execution to specific times of the day instead of disabling alerts.
https://elastalert2.readthedocs.io/en/latest/ruletypes.html#limit-execution
